docs/source/guides/configuration.rst

```diff
@@ -38,6 +40,11 @@ Deployment-specific parameters
   username, password and database name.
   All data except the database name are mandatory.
   If database name is not provided, a default one gets used.
+- ``working database`` is the name of working database - the one that will be used to store the world state view and optionally blocks.
+    .. warning:: Deprecated behaviour: is overrided by ``dbname`` parameter in ``pg_opt`` value.
+    .. warning:: Deprecated: optional.
```

Comment:
Not clear what is deprecated ((((((((((((
Optionality is deprecated and the config parameter will become required.
The parameter itself is deprecated and going to be removed.